subworkflows/local/profiling.nf

```diff
@@ -202,8 +202,8 @@ workflow PROFILING {
 
         ch_input_for_metaphlan3 = ch_input_for_profiling.metaphlan3
                             .filter{
-                                if (it[0].is_fasta) log.warn "[nf-core/taxprofiler] MetaPhlAn3 currently does not accept FASTA files as input. Skipping MetaPhlAn3 for sample ${it[0].id}."
-                                !it[0].is_fasta
+                                if (it[0].is_fasta || it[0].instrument_platform == 'OXFORD_NANOPORE' ) log.warn "[nf-core/taxprofiler] MetaPhlAn3 currently does not accept FASTA files as input and/or has not been evaluated for Nanopore data. Skipping MetaPhlAn3 for sample ${it[0].id}."
```

Comment:
Why not split this into two conditions? They seem to be quite different messages.